StokedSportsWebApi/wwwroot/index.js

```diff
@@ -1,18 +1,44 @@
-﻿
-async function sendAuthorizedRequestAsync(apiUrl, methodType, data) {
+﻿let response = await sendUnauthorizedRequestAsync("../api/Identity/Users", "POST", {})
+});
+function login(event) {
+    //store the username
+    //store password
+    //validate username and password
+    //send request to server to get a token
+    //take user to secure page
+
+    let email = document.getElementById("txtEmail").value;
+    let password = document.getElementById("txtPassword").value;
+}
+function validatePassword(password) {
+    var password = /^(?=.*[0-9])(?=.*[!@#$%^&*])[a-zA-Z0-9!@#$%^&*]{7,15}$/;
```

Comment:
In validatePassword. Return True if password is good and meets criteria. Return False if it is not good.

Reply:
Suggestion on checking password 
let passwordLength = password.length
if(passwordLength < 8) { return false }
\